addons/binding/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/Activator.java

```diff
@@ -0,0 +1,41 @@
+package org.openhab.binding.amazondashbutton.internal;
+
+import org.eclipse.smarthome.config.core.ConfigDescriptionProvider;
+import org.eclipse.smarthome.core.common.osgi.ServiceBinder;
+import org.openhab.binding.amazondashbutton.internal.config.AmazonDashButtonConfigDescriptionProvider;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceRegistration;
+
+/**
+ * The {@link Activator} is responsible building a {@link ServiceBinder} which binds the
+ * {@link AmazonDashButtonConfigDescriptionProvider}.
+ * 
+ * @author Oliver Libutzki - Initial contribution
+ *
+ */
+public class Activator implements BundleActivator {
```

Comment:
could you remove the activator and rather declare your AmazonDashButtonConfigDescriptionProvider through DS?